src/tokenizer.rs

```diff
@@ -49,6 +49,8 @@ pub enum Token {
     Char(char),
     /// Single quoted string: i.e: 'string'
     SingleQuotedString(String),
+    /// Double quoted string: i.e: "string"
```

Comment:
```suggestion
    /// Double quoted string: i.e: "string" (mysql dialect)
```

Reply:
It is also worth noting that in ANSI SQL, double quotes are used for identifiers and not for strings. For example, `monthly_income * 12 AS "Annual Income"`.